src/node/gov/handlers/service_state.h

```diff
@@ -466,31 +466,27 @@ namespace ccf::gov::endpoints
 
             auto jwt_keys_handle =
               ctx.tx.template ro<ccf::JwtPublicSigningKeys>(
-                ccf::Tables::JWT_PUBLIC_SIGNING_KEYS);
-            auto jwt_key_issuers_handle =
-              ctx.tx.template ro<ccf::JwtPublicSigningKeyIssuer>(
-                ccf::Tables::JWT_PUBLIC_SIGNING_KEY_ISSUER);
+                ccf::Tables::JWT_PUBLIC_SIGNING_KEYS_METADATA);
 
             jwt_keys_handle->foreach(
-              [&keys, jwt_key_issuers_handle](
-                const ccf::JwtKeyId& kid, const ccf::Cert& cert) {
-                auto key_info = nlohmann::json::object();
+              [&keys](
+                const ccf::JwtKeyId& k, const std::vector<KeyMetadata>& v) {
+                auto keys_info = nlohmann::json::array();
+                for (const auto& metadata : v)
+                {
+                  auto info = nlohmann::json::object();
 
-                // cert is stored as DER - convert to PEM for API
-                const auto cert_pem = crypto::cert_der_to_pem(cert);
-                key_info["certificate"] = cert_pem.str();
+                  // cert is stored as DER - convert to PEM for API
```

Comment:
Why is cert stored as DER?

Reply:
I don't know why the specific format was chosen, this comment was here before.